.appveyor.yml

```diff
@@ -39,7 +39,7 @@ install:
     - cmd: appveyor-retry conda.exe update --yes --quiet conda
 
 
-    - cmd: appveyor-retry conda.exe install --yes --quiet conda-forge-pinning conda-forge-ci-setup=1.* networkx conda-build>=3.16
+    - cmd: appveyor-retry conda.exe install --yes --quiet "conda!=4.6.1" conda-forge-pinning conda-forge-ci-setup=2.* networkx conda-build>=3.16
```

Comment:
Can you remove the "conda!=4.6.1" bits until we try it with the proper version of ci-setup?

Reply:
@scopatz, the current Travis CI builds for staged-recipes are failing with conda-4.6.1, even with `conda-forge-ci-setup=2.*`.

Reply:
Hmm do you have a link to a build that shows that?  I reallt just don't like pinning in this way, because we have to undo it later.

Reply:
@scopatz, https://travis-ci.org/conda-forge/staged-recipes/builds/485851196

Reply:
No avoiding it then, let's try it!